.upptimerc.yml

```diff
@@ -77,9 +77,17 @@ sites:
   - name: HURUmap
     url: https://hurumap.org
     icon: https://hurumap.org/static/img/icons/favicon.ico
+    expectedStatusCodes:
+      - 200   
+      - 201   
+      - 301   
```

Comment:
We expectedStatusCodes here i.e. don't think it's a problem if hurumap.org redirects.